subgraphs/_reference_/src/sdk/protocols/lending/manager.ts

```diff
@@ -227,6 +230,10 @@ export class DataManager {
     return this.market;
   }
 
+  isNewMarket(): boolean {
```

Comment:
Do you feel its better to have a method for this? I am considering its better to make the field `isNewMarket` or something and just access the field directly to reduce the amount of code needed since its just returning the value.

Reply:
AFAIK it is better OOP practice to use private variables and encapsulate when possible. And write functions to modify/access those private vars. https://stackoverflow.com/questions/14399929/should-i-use-public-or-private-variables